velox/common/memory/MmapAllocator.cpp

```diff
@@ -267,7 +267,7 @@ bool MmapAllocator::allocateContiguousImpl(
       // We failed to grow by 'newPages. So we record the freeing off the whole
       // collateral and the unmap of former 'allocation'.
       reservationCB(AllocationTraits::pageBytes(totalCollateralPages), false);
-      throw;
+      std::rethrow_exception(std::current_exception());
```

Comment:
@xiaoxmeng Good catch!
Do we have other places where we use "throw;" to rethrow?

Reply:
Yes, we do in AsyncDataCache and few others in Velox but haven't checked presto cpp. I will make a followup change for the other places. This change is exactly one of the instances you showed today offline. Thanks!